deploy/docker-compose/README

```diff
@@ -0,0 +1,153 @@
+# MONAI Deploy Lite
+
+MONAI Deploy Lite provides a simple & lightweight solution for running MONAI Applications (MAPs) on a minimal MONAI Deploy platform.
+
+It is meant to bridge the big gap between local development of MAPs in a laptop using the MONAI Deploy App SDK, and a production-ready deployment. 
+
+Users of MONAI Deploy Lite will be able to run their MAPs, connected to a test PACS, or their own test / research PACS, for further validation, confidently taking steps towards production.   
```

Comment:
NIT: remove "test". Again, we don't want to limit what DICOM devices users want to use. Clinical PACS is perfectly fine if they can de0id the PII or have the MD Lite within a security enclave.

Reply:
@dbericat ?

Reply:
You'd want to mention technical reasons behind it, so that users can make their own decision based on risk analysis. Deploy Lite does not have sufficient security measures to protect patient privacy data. If the users has means to de-id the patient info on exporting DICOM from clinical PACS, it is upto them to directly send the DICOM studies to Deploy Lite.
We provide the tool with clear warnings, and it is the user who decides how best to use it considering the inherent risks.

Reply:
I think the approach from @MMelQin makes sense.

Reply:
Agreed. Tried to capture it on my review comment as well:

"As described on the MONAI Operating Environments guideline, the journey from development to production usually requires multiple steps across different environments, operated by different teams and with different requirements.

MONAI Deploy Lite is designed to facilitate the testing and validation of MAPs in the early stages of this pipeline (i.e. workstation environment), where ease of use and time to get started is most important."

Also added some more context about enterprise features missing on https://github.com/Project-MONAI/monai-deploy/discussions/71#discussioncomment-3647885

Reply:
Updated with @dbericat's feedback